docs(mullvad): add notice about Mullvad removing OpenVPN support (#103)
qdm12/gluetun-wiki: setup/providers/mullvad.md

      - SERVER_CITIES=Amsterdam
```

> ## IMPORTANT
> 
> Mullvad will [remove OpenVPN support](https://mullvad.net/en/blog/removing-openvpn-15th-january-2026) on **15th January 2026**. It is strongly advised to start using WireGuard.

Reviewer: ```suggestion
⚠️  [Openvpn will be removed by Mullvad in 2026](https://mullvad.net/en/blog/removing-openvpn-15th-january-2026) hence the openvpn example has been removed from here.
```

- `OPENVPN_USER` which is your Mullvad user ID.

> ### WireGuard only

Reviewer: I'll batch replace this

```suggestion
### Wireguard only
```

> - `WIREGUARD_PRIVATE_KEY` is your 32 bytes key in base64 format. The private key can only be obtained by [generating a WireGuard configuration file](https://mullvad.net/en/account/#/wireguard-config). The `WireGuard Key` displayed on the [Manage devices and ports](https://mullvad.net/en/account/#/ports) page on Mullvad is *not* the private key that is required. Generate a WireGuard configuration file, download the zip file, extract any of the `.json` files within and use the `PrivateKey` value. Note this value is the same for all Mullvad servers. 💁 [Screenshots on how to obtain it](https://github.com/qdm12/gluetun/discussions/805#discussioncomment-2026642).
> - `WIREGUARD_ADDRESSES` is the WireGuard IP network interface address in CIDR format `xx.xx.xx.xx/xx`. To obtain it, first download a WireGuard configuration file using same steps as for `WIREGUARD_PRIVATE_KEY` above. In the configuration file, locate the `Address` value. This one should contain a comma delimited list of an IPv4 and IPv6 address, so use the IPv4 address (usually the first one) as the value for the `WIREGUARD_ADDRESSES` environment variable. Note this is the same value for all Mullvad servers and for your private key. 💁 [Screenshots on how to obtain it](https://github.com/qdm12/gluetun/discussions/805#discussioncomment-2026642).

Reviewer: ```suggestion
- `WIREGUARD_ADDRESSES` is the Wireguard IP network interface address in CIDR format `xx.xx.xx.xx/xx`. To obtain it, first download a Wireguard configuration file using same steps as for `WIREGUARD_PRIVATE_KEY` above. In the configuration file, locate the `Address` value. This one should contain a comma delimited list of an IPv4 and IPv6 address, so use the IPv4 address (usually the first one) as the value for the `WIREGUARD_ADDRESSES` environment variable. Note this is the same value for all Mullvad servers and for your private key. 💁 [Screenshots on how to obtain it](https://github.com/qdm12/gluetun/discussions/805#discussioncomment-2026642).
```

### Wireguard only

> - `WIREGUARD_ADDRESSES` is the Wireguard IP network interface address in CIDR format `xx.xx.xx.xx/xx`. To obtain it, first download a Wireguard configuration file using same steps as for `WIREGUARD_PRIVATE_KEY` above. In the configuration file, locate the `Address` value. This one should contain a comma delimited list of an IPv4 and IPv6 address, so use the IPv4 address (usually the first one) as the value for the `WIREGUARD_ADDRESSES` environment variable. Note this is the same value for all Mullvad servers and for your private key. 💁 [Screenshots on how to obtain it](https://github.com/qdm12/gluetun/discussions/805#discussioncomment-2026642).

Reviewer: ```suggestion
- `WIREGUARD_PRIVATE_KEY` is your 32 bytes key in base64 format. The private key can only be obtained by [generating a Wireguard configuration file](https://mullvad.net/en/account/#/wireguard-config). The `Wireguard Key` displayed on the [Manage devices and ports](https://mullvad.net/en/account/#/ports) page on Mullvad is *not* the private key that is required. Generate a Wireguard configuration file, download the zip file, extract any of the `.json` files within and use the `PrivateKey` value. Note this value is the same for all Mullvad servers. 💁 [Screenshots on how to obtain it](https://github.com/qdm12/gluetun/discussions/805#discussioncomment-2026642).
- `WIREGUARD_ADDRESSES` is the Wireguard IP network interface address in CIDR format `xx.xx.xx.xx/xx`. To obtain it, first download a Wireguard configuration file using same steps as for `WIREGUARD_PRIVATE_KEY` above. In the configuration file, locate the `Address` value. This one should contain a comma delimited list of an IPv4 and IPv6 address, so use the IPv4 address (usually the first one) as the value for the `WIREGUARD_ADDRESSES` environment variable. Note this is the same value for all Mullvad servers and for your private key. 💁 [Screenshots on how to obtain it](https://github.com/qdm12/gluetun/discussions/805#discussioncomment-2026642).
```
